### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#370] [Announcements]
We’re using Discussions as a place to connect with other members of our community. 

We hope that you will:
•	Ask questions
•	Share ideas
•	Engage with other community members
•	Welcome others
•	Be open-minded

Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us what you want to accomplish as a member of this community.

And then…happy computing 😊

P.S. Please, have also a look at [Code of Conduct](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/21299201/OpenRadioss+Code+of+Conduct) and [Acceptable Use Policy](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/21757967/Acceptable+Use+Policy)

**Comment by rsnvprasad** [comment]
Hello New Family,

I am Prasad Reddi from Bengaluru, India, practicing FEA for the past 16years using variety of Implicit & Explicit solvers and using Radioss for the last 1 year. Looking forward to use this new initiative from Altair and contribute as far as I can diversely. I am glad to be part of this community. 

Regards,
Prasad Reddi

**Reply by enghorbani** [reply]
Hello, I hope you are keeping well.
I want to learn how I can perform simulations using Radioss. Could you please help me? I need to find an example that explains step-by-step the simulation procedure using Radioss.
Thank you in advance for your help.

**Comment by Jingchao-Remy** [comment]
Welcome! Bienvenu! 欢迎！:)

**Reply by MarianBulla** [reply]
A very warm welcome, Jingchao! 
:-)

**Comment by Heguo-Ou** [comment]
Hello, old friends, I am very excited to know the GREAT FEA explicit solver - RADIOSS is opening for public. As a fan and former technical support, I can't wait to see my old friends here, and to know the latest developement.

Best wishes for you and the great solver!

**Reply by MarianBulla** [reply]
Hello, @Heguo-Ou and a very warm welcome :-)
Happy coding and using OpenRadioss!

**Comment by enghorbani** [comment]
Hello, I am a PhD student from Pusan National University. I want to perform a simulation using Radioss but I did not find an example to help me to learn step-by-step. Could you please help me with how I can start it?

**Reply by MarianBulla** [reply]
Hello @enghorbani,
thank you for your interest in OpenRadioss and a warm welcome to the community.

For the 1st steps, I would recommend you to review the YouTube videos: https://www.youtube.com/@OpenRadiossCommunity

Then, you have the documentation page at our website: www.openradioss.org

There is also a large amount of info, here: https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/4816906/OpenRadioss+User+Documentation


Hope it helps you ?

Thanks and best regards,
Marian

**Comment by CharlieM-57** [comment]
H'Yall,
I am trying to install and run OpenRadioss and it's gui. Not having much success. Any help is appreciated. Can't even get anything to start.
thanks for your help.
![06Apr26 trying to install OpenRadioss and gui](https://github.com/user-attachments/assets/39827512-d8bf-4986-b75e-7536ac950ec3)

Charlie
